### PR TITLE
updated README to clarify which of Wordpress' keys/salts to use

### PR DESCRIPTION
**Pull request by pembo13**
In reference to Issue #7 I've updated the `README.rst`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -37,6 +37,18 @@ Installation
 
 Add your WordPress's auth keys and salts (found in wp-config.php) to your settings.py.
 
+The `WORDPRESS_LOGGED_IN_KEY` value comes from your Wordpress installations
+
+.. sourcecode:: php
+
+    define('LOGGED_IN_KEY',    'rs&^D%jPdu=vk|VVDsdfsdgsdgsdg9sd87f98s7h[Xm$3gT/@1xdas');
+
+while `WORDPRESS_LOGGED_IN_SALT` value comess from
```
Comment (dellis23) [inline README.rst]: Minor thing here, but "comes" is misspelled.

**Comment by dellis23** [comment]
Thanks for this!  Just a minor spelling update and then I'll be ready to merge it.

**Comment by pembo13** [comment]
Sorry, that's more than a minor error, it's been fixed.

**Comment by dellis23** [comment]
Thanks :)

**Pull request merged**